src/main/webapp/WEB-INF/jsp/preview.jsp

```diff
@@ -66,7 +66,7 @@
             <div class="col-sm-12">
                 <c:if test="${showConfigLink}">
                     <portlet:renderURL var="configUrl" portletMode="CONFIG"/>
-                    <a class="btn btn-primary pull-right" role="button" href="${ configUrl }"><i class="fa fa-gear" aria-hidden="true"></i> <spring:message code="preview.configure"/></a>
+                    <a class="btn btn-primary pull-right" role="link" href="${ configUrl }"><i class="fa fa-gear" aria-hidden="true"></i> <spring:message code="preview.configure"/></a>
```

Comment:
:thinking: interesting :thinking:
that happens with `role="button"` or role-less `<a>` but not with `role="link"`? and only on mobile devices?